### PR TITLE
Added unbiased competition model and live script

### PR DESCRIPTION
**Pull request by smrqz**
Added code for the unbiased competition model and a draft live script.

### CODE REVIEW

File: demos/examples/UnbiasedCompetitionPNAS2019_tutorial/NWepanechnikovKernelRegrRaster.m [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+%  Nadaraya-Watson smoothing kernel regression of a rastergram using a Epanechnikov kernel
+
+  if nargin < 6
+    flag_interp = 0; % set this to 1 if you want to interpolate to the previous sampling frequency 1/(dsfact*dt)
+  end
+
+  if isempty(pop)
+    pop = unique(raster(:,2));
+  end
```
Comment (vijayiyer05) [inline demos/examples/UnbiasedCompetitionPNAS2019_tutorial/NWepanechnikovKernelRegrRaster.m:11]: You could consider using more recent [function argument validation](https://es.mathworks.com/help/matlab/matlab_prog/function-argument-validation-1.html#:~:text=A%20validation%20function%20is%20a%20MATLAB%20function%20that,the%20value%20of%20the%20arguments%20they%20are%20validating) syntax

File: demos/examples/UnbiasedCompetitionPNAS2019_tutorial/getPoissonDutyCycleGating.m [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+  if nargin<7,  widthSigma = 0.001;       end % 0.001 represents an abrupt connectivity transition (in contrast to 0.1; it only applies to dc+ac not the baseline)
+  if nargin<8,  baseline = 0;             end % Hz
+  if nargin<9,  dc_ = 0;                  end % Hz
+  if nargin<10, ac_ = 0;                  end % Hz
+  if nargin<11, slopeDutyCycle_ = 1;      end % in ms
+  if nargin<12, thresholdDutyCycle_ = 0;  end
+  if nargin<13, tau = 2;                  end
+  if nargin<14, kick = 1;                 end
+  if nargin<15, T = (0:.01:1000)';        end
+  if nargin<16, N = 1;                    end
```
Comment (vijayiyer05) [inline demos/examples/UnbiasedCompetitionPNAS2019_tutorial/getPoissonDutyCycleGating.m:18]: [Function argument validation](https://es.mathworks.com/help/matlab/matlab_prog/function-argument-validation-1.html) can help managing optional and name-value arguments.

**Comment by vijayiyer05** [comment]
Something to consider (with @jasonsherfey) is whether it makes sense to consolidate code for one tutorial (e.g. unbiased competition) into one file, either as a script file [with functions](https://www.mathworks.com/help/matlab/matlab_prog/local-functions-in-scripts.html) or a class file [with methods](https://www.mathworks.com/help/matlab/class-definition-and-organization.html).

File: demos/examples/UnbiasedCompetitionPNAS2019_tutorial/getPoissonDutyCycleGating.m (hunk 2 above)
Comment (jsherfey) [reply demos/examples/UnbiasedCompetitionPNAS2019_tutorial/getPoissonDutyCycleGating.m:18]: This is a good suggestion for future iterations and models.

**Pull request merged**